Felix45/StackOverflow-LiteApi
# Ch application structure 162879035

**What does this pull request do**
- Sets up the directory and file structure for StackOverflow-LiteApi

**Description of the task to be completed**
Sets up the directory and file structure

**How should this be manually tested**

- Clone or fork the repository 
- Open a terminal
- Change your working directory to the project directory
- Checkout to ch-application-structure-162879035
- Enter **ls -al**  command into the terminal to see the directory structure
![application-structure](https://user-images.githubusercontent.com/16392046/50533912-12594500-0b45-11e9-8313-6ffdab07b872.png)
